spock-core/src/main/java/org/spockframework/runtime/model/SpecInfo.java

```diff
@@ -126,6 +126,16 @@ public boolean getIsBottomSpec() {
     return subSpec == null;
   }
 
+  public List<SpecInfo> getSpecsToBottom() {
```

Comment:
To make the difference to `getSpecsTopToBottom` more clear, I think we should name this `getSpecsCurrentToBottom` I considered `getSpecsDescendents` but this might imply that it contains all descendants, i.e., all inheritors not just the one in this hierarchy.